# Update the canonical eigen with the non-canonical eigen_archive.

diff --git federated_language_jax/executor/BUILD federated_language_jax/executor/BUILD
index 1105719..997cbaf 100644
--- federated_language_jax/executor/BUILD
+++ federated_language_jax/executor/BUILD
@@ -208,7 +208,7 @@ cc_test(
         ":xla_utils",
         "@abseil-cpp//absl/status",
         "@abseil-cpp//absl/status:statusor",
-        "@eigen",
+        "@eigen_archive//:eigen3",
         "@federated_language//federated_language/proto:array_cc_proto",
         "@federated_language//federated_language/proto:computation_cc_proto",
         "@federated_language//federated_language/proto:data_type_cc_proto",
